Plutarch/DataRepr/Internal.hs

```diff
@@ -108,29 +80,61 @@ import Plutarch.DataRepr.Internal.HList (
 import Plutarch.Integer (PInteger)
 import Plutarch.Internal.Generic (PCode, PGeneric, gpfrom, gpto)
 import Plutarch.Internal.Lift (pconstant)
+import Plutarch.Internal.Newtype (PlutusTypeNewtype)
+import Plutarch.Internal.Other (POpaque, popaque, pto)
+import Plutarch.Internal.PLam (plam)
 import Plutarch.Internal.PlutusType (
   DerivePlutusType (DPTStrat),
   DerivedPInner,
+  PlutusType (PInner, pcon', pmatch'),
   PlutusTypeStrat,
   PlutusTypeStratConstraint,
   derivedPCon,
   derivedPMatch,
-  pcon',
-  pmatch',
+  pcon,
+  pmatch,
+ )
+import Plutarch.Internal.Term (
+  Dig,
+  Term,
+  pdelay,
+  perror,
+  pforce,
+  phoistAcyclic,
+  plet,
+  (#),
+  (#$),
+  (:-->),
+ )
+import Plutarch.Internal.Term qualified as P
+import Plutarch.Internal.TermCont (
+  TermCont,
+  hashOpenTerm,
+  runTermCont,
+  tcont,
+  unTermCont,
  )
 import Plutarch.List (PListLike (pnil), pcons, pdrop, phead, ptail, ptryIndex)
 import Plutarch.Reducible (NoReduce, Reduce)
 import Plutarch.Show (PShow (pshow'))
 import Plutarch.String (PString)
 import Plutarch.Trace (ptraceInfoError)
-import Plutarch.TryFrom (PSubtype', PSubtypeRelation (PNoSubtypeRelation, PSubtypeRelation), PTryFrom, PTryFromExcess, ptryFrom, ptryFrom', pupcast)
+import Plutarch.TryFrom (
+  PSubtype',
+  PSubtypeRelation (PNoSubtypeRelation, PSubtypeRelation),
+  PTryFrom,
+  PTryFromExcess,
+  ptryFrom,
+  ptryFrom',
+  pupcast,
+ )
 import Plutarch.Unit (PUnit (PUnit))
 import Plutarch.Unsafe (punsafeCoerce)
 
 {- | A "record" of `exists a. PAsData a`. The underlying representation is
  `PBuiltinList PData`.
 -}
-data PDataRecord (as :: [PLabeledType]) (s :: S) where
+data PDataRecord (as :: [PLabeledType]) (s :: P.S) where
```

Comment:
I prefer no quantification for `S` as per other modules

Reply:
Unavoidable here unfortunately - this module uses the `S` from `GHC.Generics` as well.